packages/cli/src/analytics/analytics.ts

```diff
@@ -72,7 +100,8 @@ export class LightdashAnalytics {
             };
 
             const body = {
-                anonymousId: uuidv4(),
+                anonymousId: user?.anonymousUuid,
```

Comment:
is the anonymousId required if userId is provided ?

Reply:
Is necessary to relate any previous events with just the anonymous uuid to a user